src/rust/engine/src/externs.rs

```diff
@@ -135,6 +135,11 @@ pub fn store_bool(val: bool) -> Value {
   with_externs(|e| (e.store_bool)(e.context, val).into())
 }
 
+#[allow(dead_code)]
+pub fn store_none() -> Value {
+  with_externs(|e| (e.store_none)(e.context).into())
```

Comment:
Does the `none()` function on line 21 of this file not do exactly this?

Reply:
Hm, I think you're right, and adding the new extern was superfluous